lib/ansible/modules/user.py

```diff
@@ -314,7 +314,7 @@
   ansible.builtin.user:
     name: james
     shell: /bin/bash
-    groups: admins,developers
+    groups: ['admins', 'developers']
```

Comment:
Personally, I'd go for a multi-line list if we're already changing this..
```suggestion
    groups:
    - admins
    - developers
```

Reply:
The original is still correct, in Ansible you can have lists in YAML, JSON and 'comma separated lists' forms.
Instead of removing it, I would add other examples with other formats. In the end it would be better to centralize such and link to a page with 'types and how to express them' page.